src/index.re

```diff
@@ -12,3 +12,21 @@ let module Window = {
 let module Console = {
   external log : 'anything => unit = "console.log" [@@bs.val];
 };
+
+let module LocalStorage = {
+  external getItem : string => 'a = "localStorage.getItem" [@@bs.val];
+  external setItem : string => 'a => unit = "localStorage.setItem" [@@bs.val];
+  external removeItem : string => unit = "localStorage.removeItem" [@@bs.val];
+  external clear : unit => unit = "localStorage.clear" [@@bs.val];
+  external key : int => 'a = "localStorage.key" [@@bs.val];
+  let getLength () => [%bs.raw {| localStorage.length |}];
+};
+
+let module SessionStorage = {
+  external getItem : string => 'a = "sessionStorage.getItem" [@@bs.val];
+  external setItem : string => 'a => unit = "sessionStorage.setItem" [@@bs.val];
+  external removeItem : string => unit = "sessionStorage.removeItem" [@@bs.val];
+  external clear : unit => unit = "sessionStorage.clear" [@@bs.val];
+  external key : int => 'a = "sessionStorage.key" [@@bs.val];
+  let getLength () => [%bs.raw {| sessionStorage.length |}];
```

Comment:
Hmm good point. We can type it as `  external length : int = "localStorage.length" [@@bs.val];` but the compiler might inline the value and produce the wrong output since `length` is mutative. @bobzhang what'd be the solution here? Maybe we need to switch back to using plain functions without module?